Bearded.Utilities.Tests/Core/interpolation/InterpolationMethod1Tests.cs

```diff
@@ -0,0 +1,29 @@
+using Bearded.Utilities.Tests.Generators;
+using FluentAssertions;
+using FsCheck.Xunit;
+
+namespace Bearded.Utilities.Tests
+{
+    public abstract class InterpolationMethod1Tests
+    {
+        protected abstract IInterpolationMethod1 Interpolation { get; }
+
+        [Property(Arbitrary = new[] {typeof(DoubleGenerators.NonInfiniteNonNaN)})]
```

Comment:
The latest version of FsCheck does no longer support registering generators in the constructor, because it no longer has static state. I could migrate this now, and it will have to be reverted to update FsCheck, so I'll leave it as is I think.